ext/MCMCChainsMakieExt.jl

```diff
@@ -0,0 +1,74 @@
+module MCMCChainsMakieExt
+
+import MCMCChains
+import Makie
+
+
+function MCMCChains.trace(chns::T) where {T<:MCMCChains.Chains}
```

Comment:
I don't think `trace` is an appropriate name, it should be something like `traceplot` or `plot_chains` to make it clear this is outputting a figure.

If you decide to go the method dispatch way, it would be good to deconstruct it a bit better:
- Make a mutable function that takes an `Axis`
- Create separate functions for each subplot.

This way the code is beneficial for anyone who would just like some part of the code.

Reply:
> I don't think trace is an appropriate name, it should be something like traceplot or plot_chains to make it clear this is outputting a figure.

`trace` follows the convention that plot names in the Julia ecosystem don't have the `plot`, e.g. `scatter`, `lines, etc.

>If convert_arguments is not sufficient, you can use the @recipe macro no?

Recipes are for creating a single sub-plot, not defining a figure with multiple sub-plots. I'm not sure there's a way to use a recipe unless all chains and parameters were plotted into a single sub-plot and not faceted the way MCMC plots tend to be

> Make a mutable function that takes an Axis

👍 

> Create separate functions for each subplot.

i.e. where you could get the left side and right sight separately?